src/amo/utils.js

```diff
@@ -1,10 +1,36 @@
-import { LANDING_PAGE_ADDON_COUNT } from 'amo/constants';
+import {
+  FEATURED_ADDONS_TO_LOAD,
+  LANDING_PAGE_ADDON_COUNT,
+} from 'amo/constants';
+import { getFeatured, loadFeatured } from 'amo/actions/featured';
 import { getLanding, loadLanding, failLanding } from 'amo/actions/landing';
 import { featured as featuredAPI, search } from 'core/api';
 import { SEARCH_SORT_POPULAR, SEARCH_SORT_TOP_RATED } from 'core/constants';
 import { apiAddonType } from 'core/utils';
 
 
+export function fetchFeaturedAddons({ addonType, api, dispatch }) {
```

Comment:
I don't understand why you need this as a separate function. Can't it just be part of `loadFeaturedAddons` ? Maybe it will become more clear in the next patch?